src/other/pages/contact/story.js

```diff
@@ -63,10 +63,10 @@ export const Contact = () => {
           </p>
           <p class="contact-support-visuals__paragraph">
             You have a digital project and
-            <strong>you want to make sure the design is compliant</strong>
-            to the AXA Brand standards? You need help and advice in adopting the
-            AXA Design System in your project? We are here to accompany you
-            through all phases.
+            <strong>want to make sure its design is compliant</strong>
+            to AXA brand standards? You need help and advice in adopting AXA's
+            Design System in your project? We are here to accompany you through
```

Comment:
`design system` instead of `Design System`?

Reply:
No, I on purpose opted for making it uppercase-initial everywhere.